docs/source/getting-started.rst

```diff
@@ -165,6 +165,8 @@ Run:
 
       npm install --save-dev mocha chai
 
+You may also need to add :code:`ts-node` and :code:`typescript` if not installed globally.
```

Comment:
Should be something more along these lines:
If you are using Typescirpt don't forget to add `ts-node` to your `devDependencies`

Reply:
Fixed. My intent here was to make it self-contained for a newcomer (i.e. going through it step by step literally should produce the described results). Even sth may be obvious to the more experienced.